spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'order'
+require 'menu'
+require 'takeaway'
+
+describe 'User Stories' do
+
+  describe 'Menu' do
+    # As a customer
+    # So that I can check if I want to order something
+    # I would like to see a menu§ of dishes with prices
+    it 'should display a list of dishes with prices' do
+      menu = Menu.new
+      expect(menu.list).not_to be_empty
+    end
+
+    it 'should allow user to add a new dish to the menu' do
+      menu = Menu.new
+      expect { menu.add_menu_item("Dish 4", 4.00) }.to change{ menu.list.count}.by(1)
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)